x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/EnrichIT.java

```diff
@@ -143,6 +144,14 @@ protected EsqlQueryResponse run(EsqlQueryRequest request) {
                 return client.execute(EsqlQueryAction.INSTANCE, request).actionGet(2, TimeUnit.MINUTES);
             } catch (Exception e) {
                 logger.info("request failed", e);
+                assertTrue(
+                    "cancellation exceptions must be ignored",
+                    ExceptionsHelper.unwrapCausesAndSuppressed(e, t -> t instanceof TaskCancelledException).isEmpty()
+                );
+                assertTrue(
+                    "remote transport exception must be unwrapped",
+                    ExceptionsHelper.unwrapCausesAndSuppressed(e, t -> t instanceof TaskCancelledException).isEmpty()
+                );
```

Comment:
It'd be nice to dump these into one place. Probably just a method you can pass the exception to....